### PR TITLE
Generated api check baselines

### PR DESCRIPTION
**Pull request by ajaybhargavb**
#212 

@dougbu 

Note the base branch is `rel/1.1.2`.

### CODE REVIEW

File: src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.DataProtection.AzureStorage, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json]: To confirm: This file was generated after building this project in the 1.0.0 branch or tag?

File: src/Microsoft.AspNetCore.DataProtection.Redis/baseline.netcore.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.DataProtection.Redis/baseline.netcore.json]: We have not released a major version of this package; highest was `0.1.2`. So, don't add a baseline and instead disable API check in the .csproj file.

File: src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json (hunk 1 above)
Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json]: It was generated after building in 1.1.2 branch.

Comment (dougbu) [inline src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json]: The baseline file should be for 1.0.0, not 1.1.2. But, it's correct check the file in here and perhaps in the `rel/1.0.3` or `patch/*` branches.

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json]: But this project was introduced in 1.1

**Comment by ajaybhargavb** [comment]
🆙 📅  Spoke to @javiercn. Looks like there is no way to disable api check in `project.json` world except not having a baseline.

File: src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json (hunk 1 above)
Comment (dougbu) [inline src/Microsoft.AspNetCore.DataProtection.AzureStorage/baseline.netcore.json]: Discussed offline&hellip; use whatever branch corresponds to the 1.0.0 version of this package.

**Pull request merged**